### PR TITLE
update template in line with documentation

### PR DESCRIPTION
**Pull request by CEKrause**
Starter kit templates updated to match documentation being drafted. 
Template version incremented to reflect the changes made in earlier updates (not so much this one, which are more cosmetic).

### CODE REVIEW

File: starter-kits/product/_overview_2.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 % ## Cite this product
-
-% > Your citation
```
Comment (benji-glitsos-ga) [inline starter-kits/product/_overview_2.md:3]: This will still need the ">" blockquote symbol because this citation should be inside a blockquote; however, maybe we should word it something like this:
```
% Note: Must use a blockquote (">"), e.g.
% > Your citation
```

File: starter-kits/product/index.rst [hunk 2]
```diff
@@ -1,2 +1,2 @@
 .. datatemplate:yaml:: _data.yaml
-   :template: product-v1.rst
+   :template: product-v2.rst
```
Comment (benji-glitsos-ga) [inline starter-kits/product/index.rst:2]: In another branch have you created a new 'v2' template? In that case, you should change this to 'v2' in your other branch and keep this as 'v1' in this branch

File: starter-kits/product/_access.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
+% - at least one of access and/or use constraints required
```
Comment (benji-glitsos-ga) [inline starter-kits/product/_access.md:1]: I am thinking that the "-" bullet points may confuse people because it is similar to the markdown syntax used in this product kit. Instead, maybe we could indicate that this is a comment by using something like:
```
% Note: At least one of access and/or use constraints required
```

Then we could do the same thing throughout, depending on what we decide.

```
% Note: Required content
```

File: starter-kits/product/_details.md [hunk 4]
```diff
@@ -1,19 +1,24 @@
 % ## Background
+% - optional
 
-% ## What this product offers
-
-% ## Data description
+% ## Product purpose
```
Comment (benji-glitsos-ga) [inline starter-kits/product/_details.md:4]: This is just a suggestion but maybe we could reword this to put the 'active' word first:
`Product purpose` => `Purpose of product`

File: starter-kits/product/_details.md [hunk 5]
```diff
@@ -1,19 +1,24 @@
...
+% - required
 
 % ## Processing steps
+% - optional
 
 % ## Software
+% - optional
 
-## References
+% ## References
```
Comment (benji-glitsos-ga) [inline starter-kits/product/_details.md:22]: The references was commented because it is an optional field. All the optional fields are commented whereas all the required fields are not commented.

File: starter-kits/product/_credits.md [hunk 6]
```diff
@@ -1,4 +1,5 @@
 % ## Acknowledgments
+% - optional, but upstream data acknowledgments, where appropriate, are required
```
Comment (benji-glitsos-ga) [inline starter-kits/product/_credits.md:2]: Will your Metadata document already tell them that something is a 'required' or 'optional' field? I am wondering if we are duplicated content here.

File: starter-kits/product/_overview_2.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
...
+% - required
 
 % ## Publications
```
Comment (benji-glitsos-ga) [inline starter-kits/product/_overview_2.md:4]: Minor thing: Since this is required, the field shouldn't be commented out.

File: starter-kits/product/_overview_2.md (hunk 1 above)
Comment (CEKrause) [reply starter-kits/product/_overview_2.md:3]: Added back.

File: starter-kits/product/_access.md (hunk 3 above)
Comment (CEKrause) [reply starter-kits/product/_access.md]: Good point. Removed.

File: starter-kits/product/index.rst (hunk 2 above)
Comment (CEKrause) [reply starter-kits/product/index.rst:2]: I'm going to scrap the other branch, so this will be v2.

File: starter-kits/product/_details.md (hunk 4 above)
Comment (CEKrause) [reply starter-kits/product/_details.md]: These words are copied from the corporate tool, so we can't change them.

File: starter-kits/product/_details.md (hunk 5 above)
Comment (CEKrause) [reply starter-kits/product/_details.md:15]: I didn't realise that was the pattern. I like that suggestion. I have copied it.

File: starter-kits/product/_credits.md (hunk 6 above)
Comment (CEKrause) [reply starter-kits/product/_credits.md]: Agreed. I have removed the text, and just used the commented or not commented as the indicator.

File: starter-kits/product/_overview_2.md (hunk 7 above)
Comment (CEKrause) [reply starter-kits/product/_overview_2.md:5]: Fixed.

**Review by benji-glitsos-ga — APPROVED** [review]
